doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -6,117 +6,397 @@ nav: /:path/Programming-guide/Key-platform-features/Endpoint-profiles/
+```
+</div>
+
+<div id="Objective-C" class="tab-pane fade" markdown="1" >
+
+```objective-c
+#import <Kaa/Kaa.h>
+#import "ViewController.h"
+ 
+@interface ViewController () <ProfileContainer>
+ 
+@property (nonatomic, strong) KAAProfile *profile;
+ 
+@end
+ 
+@implementation ViewController
+- (void)initClient() {
+    // Create instance of Kaa client
+    id<KaaClient> client = [Kaa client];
+    // Sample profile that is an auto-generated class based on user defined schema.
+    _profile = [[KAAProfile alloc] initWithId:@"id" os:OS_LINUX code:@"3.17" build:@"0.0.1-SNAPSHOT"];
+    // Simple implementation of ProfileContainer interface that is provided by the SDK
+    [client setProfileContainer:self];
+    // Starts Kaa
+    [client start];
+    // Update to profile variable
+    _profile.build = @"0.0.1-SNAPSHOT";
+    // Report update to Kaa SDK. Force delivery of updated profile to server.
+    [client updateProfile];
+}
+ 
+- (KAAProfile *)getProfile {
+    return _profile;
+}
+ 
+...
+ 
+@end
+
+```
+</div>
+</div>
 
+### Setting client-side endpoint profile schema from Admin UI
 
-![ServerSideEndpointProfileGeneration](ServerSideEndpointProfileGeneration_0_8_0.png "ServerSideEndpointProfileGeneration")
+The list of client-side endpoint profile schemas created by a tenant developer for the application is shown in the **Client-side EP profile** schemas window, which can 
+be opened from the application menu on the navigation panel as illustrated by the following screenshot.
 
+<img src="admin-ui/Client-side endpoint profile schema.png">
 
-## Server-side endpoint profile update ##
-Server-side endpoint profile is designed to be accessed and modified by various server-side applications through Kaa REST API integration layer. Server-side profile is not accessible from endpoint SDK or other client application logic.
+To export the client-side EP profile schema, click **Export** in the last column of desired schema row and select export method from drop-down. 
+See [CT schema export support]({{root_url}}Programming-guide/Key-platform-features/Common-Type-Library/#ct-schema-export-support) for available export methods details.
+As a tenant developer, you can create new client-side EP schemas for the application as follows:
+
+1. In the **Client-side EP profile schemas** window for the application, click **Add schema**.
+2. In the **Add profile schema** window enter the name of the schema.
+3. Then create a schema using one of the two options:
+    1. Using the existing CT by clicking **Select existing type** and selecting exiting CT version from FQN and version drop-downs.
```

Comment:
According to the Documentation contribution guide, this list should be of bullet-type rather than enumerated.
Also, the enumeration looks broken for the second-level list.